android/app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">Integreat</string>
+    <string name="app_name">Integreat (RN)</string>
```

Comment:
Why RN?

Reply:
So we can distinguish it from the current app.